app/Character.php

```diff
@@ -261,6 +261,13 @@ public function wishlist() {
         return $query;
     }
 
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function unReceiveditems() {
+        return $this->hasMany(CharacterItem::class)->where('is_received', 0);

```

Comment:
This is going to get prios, wishlists, and recipes. Perhaps you intend to only get wishlist or prio items?

Reply:
Thanks for the heads-up, was blindsighted because of the way we utilize TMB but it's actually a good adition to the addon as well to be able to handle both wishlisted en assigned (prio) items. Fixed!